packages/plugins/Traderxyz/src/SNSAdaptor/PostPreview.tsx

```diff
@@ -0,0 +1,313 @@
+import { Box, Chip, Grid, Typography, useTheme } from '@mui/material'
+import { makeStyles, useCustomSnackbar } from '@masknet/theme'
+import { useI18N } from '../locales/i18n_generated'
+import { usePluginWrapper } from '@masknet/plugin-infra/content-script'
+import { useAccount, useChainId, useGasLimit, useGasPrice } from '@masknet/web3-shared-evm'
+import { useTraderApi } from '../apis/nftswap'
```

Comment:
It's not a common usage, use camelCase name instead.
```suggestion
import { useTraderApi } from '../apis/nftSwap'
```

Reply:
Fixed